src/notifications/emails.py

```
@@ -0,0 +1,119 @@
+
+        Raises:
+            BaseEmailError: If sending the email fails.
+        """
+        message = MIMEMultipart()
+        message["From"] = self._email
+        message["To"] = recipient
+        message["Subject"] = subject
+        message.attach(MIMEText(html_content, "html"))
+
+        try:
+            smtp = aiosmtplib.SMTP(hostname=self._hostname, port=self._port, start_tls=self._use_tls)
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)